client/curriculum/intro/vs-code.tsx

```diff
@@ -11,6 +11,7 @@ import Lesson, {
 } from '~/components/lessons';
```

Comment:
Same as in the previous files.

We should only modify where to replace with `<FormattedText>`.